api/controllers/HomeController.js

```diff
@@ -8,7 +8,8 @@
 module.exports = _.mapValues({
   async index (req, res) {
     const boxes = await Box.find({owner: req.user.name});
-    res.view('home/view', {boxes});
+    const prefs = await UserPreferences.find({user: req.user.name});
```

Comment:
Yeah, on reconsideration I think this might the better way, provided that we don't block the page load while waiting for all the requests to come in. (For example, we can render the page without knowing the user's default upload preferences.)

I suppose another way to do it would be to have batch endpoints for all the resources one would need for rendering (to keep the number of requests down) and then test those as well to make sure they don't leak info. I feel like that still duplicates a lot of functionality though, and it increases the attack surface.